src/jit/SimdInl.h

```diff
@@ -177,8 +201,16 @@ static void emitSplatSIMD(sljit_compiler* compiler, Instruction* instr)
     JITArg args[2] = { operands + 0, operands + 1 };
     sljit_s32 dstReg = GET_TARGET_REG(args[1].arg, SLJIT_FR0);
 
+#if (defined SLJIT_64BIT_ARCHITECTURE && SLJIT_64BIT_ARCHITECTURE)
     sljit_emit_simd_replicate(compiler, type, dstReg, args[0].arg, args[0].argw);
-
+#else /* !SLJIT_64BIT_ARCHITECTURE */
+    if (type == (SLJIT_SIMD_REG_128 | SLJIT_SIMD_ELEM_64)) {
+        sljit_emit_fop1(compiler, SLJIT_MOV_F64, SLJIT_FR0, 0, args[0].arg, args[0].argw);
+        sljit_emit_fop1(compiler, SLJIT_MOV_F64, SLJIT_FR1, 0, args[0].arg, args[0].argw);
```

Comment:
Use simd mov to dstReg again. And don't load data twice, just copy dstReg to getHighRegister(dstReg).